backend/config/urls.py

```diff
@@ -3,7 +3,13 @@
 from django.contrib import admin
 from django.urls import path
 
+from .jwt_tokens import jwt_token_obtain, jwt_token_refresh
```

Comment:
It is better to avoid relative imports as they can be messy and not readable.

Reply:
fixed